5.0-SNAPSHOT build failure for Europe/London

Building from latest on main branch gives an exception on MacOS 11.5, Java 11 

I get
```
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   ToTimestampTzIntegrationTest.testColumn:72->fromEpoch:147 » DateTime Invalid v...
[ERROR]   ToTimestampTzIntegrationTest.testLiteral:109->fromEpoch:147 » DateTime Invalid...
[ERROR]   ToTimestampTzIntegrationTest.testParameter:130->fromEpoch:147 » DateTime Inval...
[INFO] 
[ERROR] Tests run: 4211, Failures: 0, Errors: 3, Skipped: 112
```

@frant-hartm thinks this can be caused by `-Duser.timezone=Europe/London`

--- a/hazelcast-sql/src/test/java/com/hazelcast/sql/impl/expression/datetime/ToTimestampTzIntegrationTest.java
+++ b/hazelcast-sql/src/test/java/com/hazelcast/sql/impl/expression/datetime/ToTimestampTzIntegrationTest.java
@@ -29,6 +29,7 @@ import org.junit.runner.RunWith;
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.time.Instant;
+import java.time.LocalDateTime;
 import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
 import java.time.temporal.ChronoUnit;
@@ -53,7 +54,10 @@ public class ToTimestampTzIntegrationTest extends ExpressionTestSupport {
     private static final Long MILLISECONDS = SECONDS * 1000L;
     private static final Long MICROSECONDS = MILLISECONDS * 1000L;
     private static final Long NANOSECONDS = MICROSECONDS * 1000L;
-    private static final Long MIN_NEGATIVE_SECONDS = -31557014135596800L;
+    private static final Long MIN_NEGATIVE_SECONDS = LocalDateTime
+            .of(-999999999, 1, 1, 0, 0)
+            .atZone(ZoneOffset.systemDefault())
+            .toEpochSecond();
 
     @Test
     public void testColumn() {

